source/common/config/utility.h

```diff
@@ -229,6 +229,15 @@ class Utility {
    */
   static std::string resourceName(const ProtobufWkt::Any& resource);
 
+  /**
+   * Envoy can have both pre-defined tag names in well_known_names and user
+   * provided tag names through bootstrap configuration. Check the confilict of
+   * all the tag names to avoid unexpected tag name overwriting. If found a
+   * confilict, throw an exception.
```

Comment:
There's a typo ("confilict"). It also might be more readable to alter the text a bit:

"Check all tag names for conflicts to avoid unexpected tag name overwriting. If a conflict is found, throw an exception."